tex2pdf_service/tex2pdf/tex2pdf_api.py

```diff
@@ -155,6 +158,15 @@ async def convert_pdf(incoming: UploadFile,
             return JSONResponse(status_code=STATCODE.HTTP_500_INTERNAL_SERVER_ERROR,
                                 content={"message": traceback.format_exc()})
 
+        if newpreflight:
+            if "newpreflight" in driver.outcome:
+                return Response(status_code=STATCODE.HTTP_200_OK,
+                                headers={"Content-Type": "application/json"},
+                                content=driver.outcome["newpreflight"])
+            else:
+                return JSONResponse(status_code=STATCODE.HTTP_500_INTERNAL_SERVER_ERROR,
+                                    content={"message": "New preflight data found"})
```

Comment:
May need to adjust error..."New preflight data found"

Reply:
Ha!! Indeed ;-) Thanks, good catch. Fixed in latest commit